Elf/__init__.py

```diff
@@ -544,7 +544,7 @@ def getPlatform(self):
             plat = e.getPlatform()
         '''
         for note in self.getNotes():
-            if note.name == 'GNU\x00' and note.ntype == 1:
+            if note.name == 'GNU' and note.ntype == 1:
```

Comment:
difference between v_bytes and v_str is whether they store the NULL byte/padding